src/gp-angular.js

```diff
@@ -597,6 +598,23 @@ module.directive('gpTranslate', ['$log', 'GlobalizationPipelineService', functio
                     if(DEBUG) {
                         $log.debug("[gp directive] promise resolved...value["+string+"]");
                     }
+
+                    // Check if the translation string has a variable expression
+                    var interpolateFn = $interpolate(string, true);

```

Comment:
No, the second parameter ensures that only strings that contain `{{stuff}}` will set a value to `interpolateFn`.  If there are no expressions to be interpolated, `interpolateFn` will be `undefined`.

So my `if` check right below will only continue if the user has passed values and the string would accept them.  If one or the other is not true, then the translated string is returned as-is.